runtime/stdlib/builtin.go

```diff
@@ -138,3 +140,97 @@ var LogFunction = NewStandardLibraryFunction(
 var HelperFunctions = StandardLibraryFunctions{
 	LogFunction,
 }
+
+var CreatePublicKeyFunction = NewStandardLibraryFunction(
+	sema.PublicKeyTypeName,
+	&sema.FunctionType{
+		Parameters: []*sema.Parameter{
+			{
+				Label:          sema.PublicKeyPublicKeyField,
+				Identifier:     sema.PublicKeyPublicKeyField,
+				TypeAnnotation: sema.NewTypeAnnotation(&sema.VariableSizedType{Type: &sema.UInt8Type{}}),
+			},
+			{
+				Label:          sema.PublicKeySignAlgoField,
+				Identifier:     sema.PublicKeySignAlgoField,
+				TypeAnnotation: sema.NewTypeAnnotation(sema.SignatureAlgorithmType),
+			},
+		},
+		ReturnTypeAnnotation:  sema.NewTypeAnnotation(sema.PublicKeyType),
+		RequiredArgumentCount: sema.RequiredArgumentCount(2),
+	},
+
+	func(invocation interpreter.Invocation) trampoline.Trampoline {
+		publicKey := invocation.Arguments[0].(*interpreter.ArrayValue)
+		signAlgo := invocation.Arguments[1].(*interpreter.CompositeValue)
+
+		value := interpreter.NewPublicKeyValue(publicKey, signAlgo)
+		return trampoline.Done{Result: value}
+	},
+)
+
+// BuiltinValues
+
+var BuiltinValues = StandardLibraryValues{
+	SignatureAlgorithmValue,
+	HashAlgorithmValue,
+}
+
+var SignatureAlgorithmValue = StandardLibraryValue{
+	Name:  sema.SignatureAlgorithmTypeName,
+	Type:  nativeEnumType(sema.SignatureAlgorithmType, sema.SignatureAlgorithms),
```

Comment:
Maybe refactor the common subexpression `nativeEnumType(sema.SignatureAlgorithmType, sema.SignatureAlgorithms)` into a variable declaration

Reply:
`nativeEnumType(sema.SignatureAlgorithmType, sema.SignatureAlgorithms)` is only used here. In the other place, arguments are different.

Reply:
Ah, I missed that! Makes sense 👍